schema/enums/QuantitySourceType.schema.json

```diff
@@ -12,5 +12,5 @@
     "INSTRUMENT_MAX",
     "INSTRUMENT_MIN"
   ],
-  "$comment": "Copyright © 2023 Open Cap Table Coalition (https://opencaptablecoalition.com) / Original File: https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/tree/main/schema/types/QuantitySourceType.schema.json"
+  "$comment": "Copyright © 2023 Open Cap Table Coalition (https://opencaptablecoalition.com) / Original File: https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/tree/main/schema/enums/quantitysourcetype.schema.json"
```

Comment:
We need to make changes where this is referenced too:

https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/fec5c3962e15735caf0607b48d3ae2484efea59e/schema/objects/transactions/issuance/WarrantIssuance.schema.json#L72-L77

Reply:
We would leave this in camel case too to conform with other schemas.

Reply:
Oh, just realizing this is just in the wrong place. The $id is perfectly fine and doesn't need to be changed.